.github/workflows/build_llvm.yml

```diff
@@ -0,0 +1,50 @@
+name: CI
+
+on:
+  push:
+    branches: [ release_90 ]
+    
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        target: [X86, AArch64, PowerPC]
+
+    steps:
+      - name: Check tools
+        run: |
+          pwd
+          git --version
+          cmake --version
+          make --version
+          gcc --version
+          
+      - name: Build llvm
+        run: |
+          # exit the llvm dir, so the path is the same on subsequent projects
+          cd ../..
```

Comment:
Question: Why does the GitHub action put the process inside `/home/runner/work/llvm/llvm` on start-up?

Reply:
Checkout action [README](https://github.com/actions/checkout) says:

> Improved layout
> * The input path is always relative to $GITHUB_WORKSPACE
> * Aligns better with container actions, where $GITHUB_WORKSPACE gets mapped in

I tried to set `path` to `../..` but got an error, so the `cd ../..` is a workaround for checkout's limitation.

Github actions are developing and growing quite quickly, so perhaps soon there will be a way to simplify this.

Reply:
Thanks @michalpasztamobica.